ddR/R/ddR.R

```diff
@@ -55,20 +56,19 @@ ddR.env$nexecutors <- 1
 #' library(distributedR.ddR); useBackend(distributedR)
 #' } 
 #' @export
-useBackend <- function(driver, ...) {
+useBackend <- function(driver = parallel, ...) {
 
-  if(!is.null(ddR.env$driver)) shutdown(ddR.env$driver)
+    # validate first or else shutdown can fail on subsequent calls
+    if(!is(driver, "ddRDriver")){
+        stop("Invalid driver object specified")
+    }
 
-  if(!extends(class(driver)[[1]],"ddRDriver")) stop("Invalid driver object specified")
+    if(!is.null(ddR.env$driver)) shutdown(ddR.env$driver)
 
-  if(!extends(driver@DListClass,"DObject")) stop("The driver DList class does not extend ddR::Dobject")
```

Comment:
The point of these checks is to protect the user from supplying an invalid driver object, i.e., he/she may type 
`useBackend(foo)`, where `foo` may be anything. We need not only check for inheritance of the driver object but also the dobject types it uses.

 I'm not opposed to having these checks in the tests, but I think that they should still be in the package code.